.gitignore

```diff
@@ -60,3 +60,5 @@ dist-ssr
 .coverage
 
 backend/README.md
+
+myenv
```

Comment:
This is specific to your dev environment and should not be in this PR!

Protip: update gitignore in your system's git config/home directory!

Reply:
😅 Agree, had to swap to my private hardware, and forgot to check that the global git config was set up.